pennylane/utils.py

```diff
@@ -340,7 +340,7 @@ def circuit2():
             if o.inverse:
                 o.inv()
 
-    tape.inv()
+    qml.transforms.invisible(tape.inv)()
```

Comment:
Why make is this instance invisible, but other instances aren't?

Reply:
Good point. `tape.inv()` previously did not affect queuing, since it was a pure in-place method. However:

- It now calls `op.adjoint()` where defined, and `op.adjoint()` 'queues'. This was causing tests where `qml.adjoint()` is used _inside_ the QNode to fail! Marking it as invisible fixed the failing tests.

- As to why the template `expand()` method _don't_ require this: these lines were added to support `qml.inv()`, which is already invisible. When `qml.inv()` is removed, these if statements can also be removed.